src/app/qgisapp.cpp

```diff
@@ -2749,6 +2750,11 @@ void QgisApp::createActions()
   mActionAddOracleLayer = nullptr;
 #endif
 
+#ifndef HAVE_HANA
```

Comment:
sometimes mActionAddHanaLayer is protected by #ifndef, sometimes it isn't. This should be harmonized

Reply:
It is protected by analogy with mActionAddPgLayer and mActionAddOracleLayer.